circleci.sh

```diff
@@ -89,7 +89,7 @@ coverage() {
 
     # rebuild dmd with coverage enabled
     # use the just build dmd as host compiler this time
-    mv src/dmd src/host_dmd
+    mv generated/dmd src/host_dmd
```

Comment:
AFAIK _a lot_ of stuff depends on `src/dmd` (e.g. all Makefiles in the `dlang` repo)

https://github.com/dlang/druntime/blob/master/posix.mak#L24
https://github.com/dlang/phobos/blob/master/posix.mak#L90
https://github.com/dlang/dlang.org/blob/master/posix.mak#L22

(and of course equivalently for Windows)